docs/index.rst

```diff
@@ -2607,7 +2607,7 @@ Running tests
 
 There are two ways of running tests. If psutil is already installed use::
 
-    $ python -m psutil.tests
+    $ python -m tests
 
 You can use this method as a quick way to make sure psutil fully works on your
 platform. If you have a copy of the source code you can also use::
```

Comment:
This whole part ("there are 2 ways of running tests") should go away as from now on there is only one.